wwwroot/about.html

```diff
@@ -294,8 +296,9 @@ <h3>Get started in 3 simple steps</h3>
                         </li>
                     </ul>
                 </div>
-                <div class="col-md-4 py-2 text-md-end">
+                <div class="col-lg py-2 text-lg-end">
                     <a href="/swagger" class="d-inline-block me-3">API</a>
+                    <a href="https://zksnacks.com/coinjoin-api" target="_blank" class="d-inline-block me-3">Integrations</a>
```

Comment:
This isn't the right link right?

Reply:
This will, but we have to wait with the merge until the zksnacks website also updates.

Reply:
Ah ok, 
maybe mark the PR as draft until then